src/components/grid-view/__tests__/GridView-test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import GridView from '../GridView';
+
+describe('components/grid-view/GridView', () => {
+    test('should render()', () => {
+        const collection = { items: [{ type: 'folder', id: '001', name: 'Example Folder' }] };
+        const wrapper = shallow(
+            <GridView
+                columnCount={5}
+                count={10}
```

Comment:
Is count the number of items in the collection? Could we derive that from the collection itself rather than set it as a prop?